patches/5454082B - Red Dead Redemption (Launch NTSC Disc).patch

```diff
@@ -1,6 +1,7 @@
 title_name = "Red Dead Redemption"
 title_id = "5454082B"
-hash = "82CDA3C91B57C170"
+#hash = "82CDA3C91B57C170"
+#hash = "BCEC4C4143B6BF5F"
```

Comment:
Only single hash is allowed per file

Reply:
Yes, I realized that before submitted, thus why I commented them out.
I was simply curious if multiple hashes could be available within one patch file.

I will make a new file.